source/import-pipeline-from-text.txt

```diff
@@ -12,7 +12,10 @@ Import Pipeline from Text
    :depth: 1
    :class: singlecol
 
-*New in version 1.15.0*
+Starting in Compass 1.35, the import pipeline button has been removed
```

Comment:
I'm not sure this is the best place for this paragraph. As a new user coming in with no context on this procedure, this could potentially read like I'm no longer able to import pipelines, especially if I read too quickly.

Rather than focusing on the negative (what's been removed), maybe this could be reworded a bit to say that the location of the import functionality has moved?

I would also consider making this a note after the intro paragraph, as brand new Compass users don't need to know this information.